### ISSUE TITLE
Add a basic stock managment system

### ISSUE THREAD

**Issue by citorva** [bde-isima/bde.isima.fr#40]
To make stock managing efficient, it is useful to add a system which manages the quantity of each articles (if specified)

This system must add these following features:
 + Add a form to add articles in the counter
 + Automatically substract items when a sell is made
 + Add a way to remove items when expiration date has been reached

**Comment by IraSkyx** [comment]
Agreed. Beware though that this was an idea we had a while back, but ended up not doing it because of the cumbersomeness of syncing virtual with real stocks.
I think some work have been started in https://github.com/bde-isima/bde.isima.fr/tree/develop_nguyen
